specification/securitydevops/resource-manager/Microsoft.SecurityDevOps/preview/2022-09-01-preview/securitydevops.json

```diff
@@ -0,0 +1,2692 @@
+        "modelAsExtensible": true
+      }
+    },
+    "AzureDevOpsConnector": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AzureDevOpsConnectorProperties"
+        }
+      }
+    },
+    "AzureDevOpsConnectorListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AzureDevOpsConnector"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "Gets or sets next link to scroll over the results.",
+          "type": "string"
+        }
+      }
+    },
+    "AzureDevOpsConnectorProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "authorization": {
+          "$ref": "#/definitions/AuthorizationInfo"
+        },
+        "orgs": {
+          "description": "Gets or sets org onboarding information.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AzureDevOpsOrgMetadata"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "AzureDevOpsConnectorStats": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AzureDevOpsConnectorStatsProperties"
+        }
+      }
+    },
+    "AzureDevOpsConnectorStatsListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AzureDevOpsConnectorStats"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "Gets or sets next link to scroll over the results.",
+          "type": "string"
+        }
+      }
+    },
+    "AzureDevOpsConnectorStatsProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "orgsCount": {
+          "format": "int64",
+          "description": "Gets or sets orgs count.",
+          "type": "integer"
+        },
+        "projectsCount": {
+          "format": "int64",
+          "description": "Gets or sets projects count.",
+          "type": "integer"
+        },
+        "reposCount": {
+          "format": "int64",
+          "description": "Gets or sets repos count.",
+          "type": "integer"
+        }
```

Comment:
I presume these are meant to be "readonly": true

Reply:
This is hopefully an API we dont need to bring to next API version once the resources are onboarded to ARG. So keep this as is.